Xamarin.Forms.Platform.UAP/SwitchRenderer.cs

```diff
@@ -28,6 +36,12 @@ protected override void OnElementChanged(ElementChangedEventArgs<Switch> e)
 			}
 		}
 
+		private void Control_Loaded(object sender, RoutedEventArgs e)
+		{
+			UpdateOnColor();
+			Control.Loaded -= Control_Loaded;
```

Comment:
not sure this is required

Reply:
You mean using the `Loaded` event? I see in the `SwitchCell` (https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Platform.UAP/CellControl.cs#L106) that it happens in the measure override. Would you rather have it there?